docs/elasticsearch/catapis.md

```diff
@@ -56,53 +56,68 @@ Typically, for any operation you can find out what headers are available using t
 {:toc}
 
 ---
+## Aliases
```

Comment:
Definitely appreciate the alphabetization. I wasn't sure what you'd done, but it makes the page easier to scan.